Chargebee/Classes/Purchase/CBPurchaseManager.swift

```diff
@@ -205,6 +205,7 @@ extension CBPurchase: SKProductsRequestDelegate {
 
 // MARK: - SKPaymentTransactionObserver delegates
 extension CBPurchase: SKPaymentTransactionObserver {
+    // swiftlint:disable:next cyclomatic_complexity
```

Comment:
Temporarily disables lint check. This will be refactored separately. cc: @cb-gadagoju